### PR TITLE
Implement cli interface

### PR DESCRIPTION
**Pull request by wreulicke**
* rewrite codes using url.Values for the form request
* should use lowerCamel as variable name
* implement cli interface

### Syukkin 

Syukkin is soooooooooooooooooooooooooooooooooooooooo easy

```bash
$ kotrcli syukkin
# kotrcli s
#
```

### Taikin

Taikin is soooooooooooooooooooooooooooooooooooooooo easy

```bash
$ kotrcli taikin
# kotrcli t
```

## Setup

You should write settings in ~/.kot/config

```
[default] # this PR is only support default section
user_token = <your-user-token>
token = <your-token>
```

### CODE REVIEW

File: kotrcli.go [hunk 1]
```diff
@@ -15,30 +17,47 @@ const (
...
+		}
+	}
 	body, err := ioutil.ReadAll(resp.Body)
 	if err != nil {
-		return "", err
+		return err
 	}
-	return string(body), nil
+	fmt.Fprintln(os.Stderr, string(body))
+	return nil
```
Comment (inabajunmr) [inline kotrcli.go:56]: thanks prrrrr!!!!! @wreulicke 
I consider about integration with AWS Lambda via HTTP for slack command.
When I create Lambda Function, I suppose to use this client as library so I think caller need kot's response body for lambda http response(for slack).
What should we do for this integration?

Comment (wreulicke) [reply kotrcli.go:56]: fmm...
OK, We should use io.Writer such as bytes.Buffer.

Comment (wreulicke) [reply kotrcli.go:56]: We can create entrypoint like `lambda/main.go` if we need executable for lambda.

example using bytes.Buffer 

```go
func Handler(.....) {
  buf := new(bytes.Buffer)
  _ := kotrcli.Dakoku(kotrcli.Syukkin, userToken, token)
  body := buf.String()
  // .............
}
```

**Comment by inabajunmr** [comment]
It works!
THANKYOUUUUUUUUUU

**Pull request merged**